src/Services/Catalog/CatalogService.Application/UseCases/Author/Queries/AuthorListVm.cs

```diff
@@ -0,0 +1,14 @@
+﻿using CatalogService.Application.Dto;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CatalogService.Application.UseCases
+{
+    public class AuthorListVm
```

Comment:
It is okay to return collection from your handlers and even endpoints, why do you create empty class that will not have any additional info?

for example you should create additional dto class with collection of entities in it when you have pagination information used to retrieve the collection